mtp_noms_ops/apps/settings/forms.py

```diff
@@ -264,3 +264,47 @@ def get_confirmation_query_string(self):
         query_dict = self.request.GET.copy()
         query_dict['prisons'] = prisons
         return urlencode(query_dict, doseq=True)
+
+
+class JobInformationForm(ApiForm):
+    job_titles = [
+        (_('Evidence collator'), 'Evidence collator'),
```

Comment:
aren't choice supposed to be value-label pairs not label-value?

Reply:
You're right - I think I got this from the only place that doesn't label them correctly, but this is a different context so possible correct too (https://github.com/ministryofjustice/money-to-prisoners-noms-ops/blob/6105043aa8a2ee4c0d2acf7b6a4b326b521c0ab9/mtp_noms_ops/apps/security/forms/object_base.py#L89-L90)

Reply:
the `.name` and `.value` come from the Enum class but we're using the name as the vale and the value as the label in the choices :)

Reply:
I see, yep, lacking context there - sorting that now though